examples/nlp/glue_with_BERT.py

```diff
@@ -178,10 +184,16 @@
     """ Use this if you're using a BERT model that you pre-trained yourself.
     Replace BERT-STEP-150000.pt with the path to your checkpoint.
     """
-    tokenizer = SentencePieceTokenizer(model_path=args.tokenizer_model)
-    tokenizer.add_special_tokens(["[MASK]", "[CLS]", "[SEP]"])
+    if args.tokenizer == "sentencepiece":
+        tokenizer = SentencePieceTokenizer(model_path=args.tokenizer_model)
+        tokenizer.add_special_tokens(["[MASK]", "[CLS]", "[SEP]"])
+    elif args.tokenizer == "nemobert":
+        tokenizer = NemoBertTokenizer(args.pretrained_bert_model)
+    else:
+        raise ValueError(f"received unexpected tokenizer '{args.tokenizer}'")
+    config = BertConfig.from_json_file(args.bert_config)
```

Comment:
if "--pretrained_bert_model" is specified then your lines 194 - 197 are not needed. Could you please change to reflect that?

Reply:
done